notebooks/end-to-end-structured/labs/4b_keras_dnn_babyweight.ipynb

```diff
@@ -11,7 +11,6 @@
     "1. Set CSV Columns, label column, and column defaults\n",
```

Comment:
<p>Same remarks as in the solution:</p><ul><li>Replace StringLookup by CategoryEncoding</li><li>Replace "cc" by a more descriptive name</li></ul><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/204/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>